core/src/test/java/io/questdb/std/IntStackTest.java

```diff
@@ -47,4 +47,27 @@ public void testStack() {
             Assert.assertEquals("at " + count, expected[count++], stack.pop());
         }
     }
+
+    @Test
+    public void testPollLast() {
+        int sz = 2050;
+        int[] expected = new int[sz];
+        int l = expected.length - 1;
+
+        IntStack stack = new IntStack();
+        for (int i = expected.length - 1; i > -1; i--) {
+            stack.push(expected[l--] = i);
+        }
+
+        Assert.assertEquals(sz, stack.size());
+
+        int count = expected.length - 1;
+        while (stack.notEmpty()) {
+            Assert.assertEquals("at " + count, expected[count--], stack.pollLast());
+        }
+
+        // The following assertions fail while they shouldn't.
```

Comment:
nit: this comment is redundant.

Reply:
removed: https://github.com/questdb/questdb/pull/2037/commits/f38e19731353ed9d2bf6ce4422741f04eabe7873